src/contexts/html.ts

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright 2017-present Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the MIT License (the "License"); you may obtain a copy of the
+ * license at https://github.com/palantir/typesettable/blob/develop/LICENSE
+ */
+
+import { IDimensions } from "../measurers";
+import { IAnchor, ITransform } from "../writers";
+import { ITypesetterContext } from "./index";
+
+export class HtmlUtils {
+  /**
+   * Appends an SVG element with the specified tag name to the provided element.
+   * The variadic classnames are added to the new element.
+   *
+   * Returns the new element.
+   */
+  public static append(element: Element, tagName: string, ...classNames: string[]) {
+    const child = HtmlUtils.create(tagName, ...classNames);
+    element.appendChild(child);
+    return child;
+  }
+
+  /**
+   * Creates and returns a new SVGElement with the attached classnames.
+   */
+  public static create(tagName: string, ...classNames: string[]) {
+    const element = document.createElement(tagName) as HTMLElement;
```

Comment:
I wonder if there's some way to proxy all the recognized built-in document.createElement() string literal types to the `tagName` so we don't have to cast